server/spec/test_integration_ldap_check.rb

```diff
@@ -68,8 +68,12 @@ def test_constructor_config_file_value_override
     assert_equal(7,@x.configuration['cache_seconds'],"override value in config file")
   end
 
-  ## test to see if the membership query finds person that is in group
+  def test_constructor_bad_group_name
+    @x = LdapCheck.new('group' => @group+"HECK_no", 'config_file' => @config_file, 'cache_seconds' => 10)
```

Comment:
The comment says the purpose is to "Deal with empty or missing LDAP group". I don't see the empty LDAP group part been tested here.